### PR TITLE
Alik front

### PR DESCRIPTION
**Pull request by aliklynnlyle**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -46,6 +46,9 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+# Enables the creation of both simple and advanced search forms
+gem 'ransack'
```
Comment (jrmcgarvey) [inline Gemfile]: You need to specify the specific level of the gem in the Gemfile.

File: app/controllers/placements_controller.rb [hunk 2]
```diff
@@ -1,6 +1,59 @@
 # frozen_string_literal: true
 
 class PlacementsController < ApplicationController
+
+#     before_action :authenticate_user!
+#     require 'roo'
```
Comment (jrmcgarvey) [inline app/controllers/placements_controller.rb]: This big section of old code now commented out should be removed.

File: app/controllers/placements_controller.rb [hunk 3]
```diff
@@ -21,12 +74,43 @@ def create
...
-    @placements = Placement.all
-    search = "%#{params[:search]}%"
-    @placements_search = Placement.where('name LIKE ? OR county LIKE ?',
-                                         search, search)
+
+
+    # search = "%#{params[:search]}%"
+    # @placements = Placement.where('name LIKE ? OR county LIKE ?', search, search)
+      
+    @q = Placement.ransack(params[:q])
```
Comment (jrmcgarvey) [inline app/controllers/placements_controller.rb]: q is not a good choice for a variable name.  You want to use something descriptive.

**Comment by aliklynnlyle** [comment]
Ok, but if my code is not being used, do I need to resolve these conflicts?
I imagine they are not using my search feature, so I am not sure it would
make sense to resolve with my code for that?

Also, the q variable was used on the Ransack gem page, so that's why I used
it.

**Comment by jrmcgarvey** [comment]
The team decided not to use these changes.

**Pull request closed without merging**